src/frontend/src/btc/stores/utxos-fee.store.ts

```diff
@@ -32,3 +34,5 @@ export interface UtxosFeeContext {
 }
 
 export const UTXOS_FEE_CONTEXT_KEY = Symbol('utxos-fee');
+
+export const utxosFeeStore = initUtxosFeeStore();
```

Comment:
Do I understand correctly that your proposed solution is to turn the utxosFeeStore into a global store instead of a context-specific store? Spontaneously, this feels like a shortcut, but I’m unsure of the implications. Is it appropriate to have UTXO fees stored globally when no wizard is open? If someone accesses this information in such cases, what would it represent or imply?

Reply:
Yes, correct, I switched it to be a global store, since I did not find any side effects or implications. True, it can be now accessed outside of the wizard, but the data stored there (`undefined` or `utxosFee` for `amount`) is not very meaningful outside of BTC conversion or send flows. Even though it would be definitely nicer to keep it scoped, the pattern I'm using in this PR is already in the codebase (other BTC/ETH/ICP stores), so I think it already proved to be working.

Reply:
Thanks for the explanation. There is no side effects, for now 😉. In the future do you foreseen any use cases that would justify having this store being available globally? If not, is there a way to resolve the issue while keep a context store?

Reply:
Cool, thanks! Added two minor comments but, looks good!